<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>615</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QSS Style Sheet Gen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>406</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>406</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>示例</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="demoBtn1">
               <property name="text">
                <string>enabled</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="demoBtn2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>disabled</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="demoBtn3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>checkable</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>406</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>图片</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="btnReadImg">
               <property name="text">
                <string>选择图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="imgPreview">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
              <property name="title">
               <string>示例</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonDemo">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowserForToolBtn">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>模板函数(注意资源中的图片路径,和文件夹中的图片路径的写法区别)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>185</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;void&lt;/span&gt;&lt;span style=&quot; color:#ff007f;&quot;&gt; init3StateBtn&lt;/span&gt;(&lt;span style=&quot; color:#0000ff;&quot;&gt;const QString&lt;/span&gt;&amp;amp; imgPath, &lt;span style=&quot; color:#0000ff;&quot;&gt;QToolButton&lt;/span&gt;* btn)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;{&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    &lt;span style=&quot; color:#0000ff;&quot;&gt;QStringList&lt;/span&gt; qss;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    qss.&lt;span style=&quot; color:#ff557f;&quot;&gt;append&lt;/span&gt;(&lt;span style=&quot; color:#0000ff;&quot;&gt;QString&lt;/span&gt;(&lt;span style=&quot; color:#00aa00;&quot;&gt;&amp;quot;QToolButton{border-style:none;background:url(:%1);background-position:left;}&amp;quot;&lt;/span&gt;).&lt;span style=&quot; color:#ff557f;&quot;&gt;arg&lt;/span&gt;(imgPath));&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    qss.&lt;span style=&quot; color:#ff557f;&quot;&gt;append&lt;/span&gt;(&lt;span style=&quot; color:#0000ff;&quot;&gt;QString&lt;/span&gt;(&lt;span style=&quot; color:#00aa00;&quot;&gt;&amp;quot;QToolButton:hover{border-style:none;background:url(:%1);background-position:center;}&amp;quot;&lt;/span&gt;).&lt;span style=&quot; color:#ff557f;&quot;&gt;arg&lt;/span&gt;(imgPath));&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    qss.&lt;span style=&quot; color:#ff557f;&quot;&gt;append&lt;/span&gt;(&lt;span style=&quot; color:#0000ff;&quot;&gt;QString&lt;/span&gt;(&lt;span style=&quot; color:#00aa00;&quot;&gt;&amp;quot;QToolButton:pressed{border-style:none;background:url(:%1);background-position:right;}&amp;quot;&lt;/span&gt;).&lt;span style=&quot; color:#ff557f;&quot;&gt;arg&lt;/span&gt;(imgPath));&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    btn-&amp;gt;&lt;span style=&quot; color:#ff557f;&quot;&gt;setStyleSheet&lt;/span&gt;(qss.&lt;span style=&quot; color:#ff557f;&quot;&gt;join&lt;/span&gt;(&lt;span style=&quot; color:#00aa00;&quot;&gt;&amp;quot;&amp;quot;&lt;/span&gt;));&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;}&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="menuCreateQPushButton"/>
    <addaction name="menuCreateQToolButton3State"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="menuCreateQPushButton">
   <property name="text">
    <string>创建QPushButton</string>
   </property>
  </action>
  <action name="menuCreateQToolButton3State">
   <property name="text">
    <string>创建三态QToolButton</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
